src/undo/undo_manager.cpp

```diff
@@ -41,7 +41,7 @@
 
 namespace OpenOrienteering {
 
-Q_STATIC_ASSERT(UndoManager::max_undo_steps < std::numeric_limits<int>::max());
+Q_STATIC_ASSERT((int)UndoManager::max_undo_steps < std::numeric_limits<int>::max());
```

Comment:
That's a type of cast we avoid in modern C++. I can't add a proposal right now. I would need to check which limit and conversion is actually guarded here.

Reply:
My initial thought was to remove that check completely. It merely checks that the value 'UndoManager::max_undo_steps' (which is actually 128) is less than the maximum number of an integer.

Reply:
Do you prefer
`Q_STATIC_ASSERT(static_cast<int>(UndoManager::max_undo_steps) < std::numeric_limits<int>::max());`
instead?